src/DXRMapFixups/DeusEx/Classes/DXRFixupM03.uc

```diff
@@ -711,10 +717,22 @@ function AnyEntryMapFixes()
             }
         }
         break;
+    case "03_NYC_UNATCOHQ":
+        // 'AnnaThianks_Played' is expired here, but 'AnnaThanksChatDone' isn't
+        // restores an alternative line from Anna about a debriefing based on what you did in M02
+        FixConversationFlagJump(GetConversation('AnnaAtUNATCO'), 'AnnaThanks_Played', false, 'AnnaThanksChatDone', false);
```

Comment:
> A debriefing.  Do not worry.  I will state in my report that I was satisfied with your performance.

Reply:
Typo in the comment, "AnnaThianks_Played" instead of "AnnaThanks_Played"